org.gnome.innstereo.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "org.gnome.innstereo",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "innstereo",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
```

Comment:
If you upgrade to 3.30 you can remove the cflags & cxxflags as they are set by default now in the FreeDesktop 18.08 (GNOME 3.30 is based on it)